### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by hakubo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,8 @@ Test screens in acceptance/integration tests for visual changes over time.
 [![Build Status](https://travis-ci.org/Cropster/ember-visual-test.svg?branch=master)](https://travis-ci.org/Cropster/ember-visual-test)
 [![npm version](https://badge.fury.io/js/ember-visual-test.svg)](https://badge.fury.io/js/ember-visual-test)
 
-## Installation
```
Comment (mydea) [inline README.md]: Could you revert that change, as it isn't really necessary and makes the code style inconsistent?

File: package.json [hunk 2]
```diff
@@ -15,37 +15,42 @@
...
-    "eslint-plugin-ember": "^5.0.3",
-    "eslint-plugin-node": "^5.2.1",
-    "loader.js": "^4.2.3"
+    "ember-load-initializers": "^1.1.0",
+    "ember-maybe-import-regenerator": "^0.1.6",
+    "ember-resolver": "^4.5.5",
+    "ember-source": "~3.1.0",
+    "ember-source-channel-url": "^1.0.1",
+    "ember-try": "^0.2.23",
+    "eslint": "^4.19.1",
```
Comment (mydea) [inline package.json]: Please remove this dependency (eslint), as it is will be automatically included by ember-cli-eslint anyhow.

Comment (hakubo) [reply package.json]: I can remove it,
but I think it's safer to list eslint here
instead of relying on other dependencies to install it

your call :)

Comment (mydea) [reply package.json]: OK, we can also leave it, I guess it doesn't matter too much!

**Comment by mydea** [comment]
Thanks for that! :)

**Pull request merged**